README.md

```diff
@@ -13,22 +13,22 @@ Clone repo via git or use the template button above.
 
 Install the dependencies
 ```
-npm install
+yarn install
```

Comment:
why yarn? 😃

Reply:
yarn is the default used by vue-cli :D , we still can use npm

Reply:
seems that it depends on `~/.vuerc`, I have `"packageManager": "npm"` there (didn't edit this file by myself).

Reply:
updated